### PR TITLE
Add anon tracking notice for installers

### LINKED ISSUE #7401: Inform users about telemetry and how to opt-out
Indicate in all installers before the Enter/yes-no: "Anonymous statistics will be [enabled/disabled]", according to the provided option. Provide information on the option relevant to the specific installer, to opt-out:

- [x] netdata-installer.sh
- [x] kickstart.sh 
- [x] kickstart-static64.sh
- [ ] binary packages (not sure about this one)

For docker images:

- [x] Add a `DO_NOT_TRACK` env var which when set will disable the anonymous stat pings.
- [x] Document the env var

Related: https://github.com/netdata/netdata/issues/7404#event-2848554127

### LINKED ISSUE #7470: kickstart-static64.sh: Unknown option '--disable-telemetry' or invalid number of arguments.
##### Bug report summary

The [pre-built static binary installation](https://github.com/netdata/netdata/tree/master/packaging/installer#linux-64bit-pre-built-static-binary) method fails when adding the option `--disable-telemetry`:

This works:

```bash
$ bash <(curl -Ss https://my-netdata.io/kickstart-static64.sh) --dont-wait --no-updates --stable-channel
```

This does not work:

```bash
$ bash <(curl -Ss https://my-netdata.io/kickstart-static64.sh) --dont-wait --no-updates --stable-channel --disable-telemetry
```

The error message:

```plain
Unknown option '--disable-telemetry' or invalid number of arguments. Please check the README for the available arguments of /dev/fd/63 and try again
```

##### OS / Environment

The environment is a fresh install of Debian 10.2 in a VM:

```
$ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

##### Netdata version (ouput of `netdata -V`)

N/A; `netdata` will not install.

##### Steps To Reproduce

Append `--disable-telemetry` to the pre-built static binary installation method:

```bash
$ bash <(curl -Ss https://my-netdata.io/kickstart-static64.sh) --dont-wait --no-updates --stable-channel --disable-telemetry
```

##### Expected behavior

No error message; successful installation.

### PR DESCRIPTION
**Pull request by ncmans**
Fixes #7401 
Fixes #7470 

## TODO

- [x] Update kickstart installers

### CODE REVIEW

**Comment by sneak** [comment]
You should be checking for DO_NOT_TRACK as non-zero and non-empty, not specifically equal to 1. 

Additionally, the claim that the statistics being collected are anonymous is false.

**Comment by OdysLam** [comment]
Hey @ncmans, 

check this: https://github.com/Homebrew/brew/pull/6745 🤔

File: packaging/docker/run.sh [hunk 1]
```diff
@@ -7,6 +7,10 @@
 # Author  : Pavlos Emm. Katsoulakis <paul@netdata.cloud>
 set -e
 
+if [ "$DO_NOT_TRACK" = 1 ]; then
```
Comment (cakrit) [inline packaging/docker/run.sh]: I think the comment on this is minor but correct. Can't we just check for existence here and  convert L64 in packaging/docker/Dockerfile to  a commented line (#ENV DO_NOT_TRACK 1)?

Comment (ncmans) [reply packaging/docker/run.sh]: Done.

**Comment by Ferroin** [comment]
Also Fixes:#7470

Somehow the docs got updated to mention the option before this got merged.

**Comment by joelhans** [comment]
Hey @Ferroin — Where do you see the docs updated? All I'm aware of is the my PR that's still open, unless someone managed to sneak some changes to the docs past me completely...

**Comment by Ferroin** [comment]
@joelhans On the live docs site, [anonymous statistics page](https://docs.netdata.cloud/docs/anonymous-statistics/), all the way at the bottom it mentions the `--disable-telemetry` option which this PR is adding.

**Pull request merged**